apps/judicial-system/web/src/routes/Prosecutor/components/PoliceCaseFiles/PoliceCaseFiles.tsx

```diff
@@ -74,128 +56,54 @@ const CheckboxList: React.FC<ListItemProps> = ({
 }) => (
   <AnimatePresence>
     {files.map((file) => (
-      <CheckboxListItem key={file.id}>
-        <Checkbox
-          label={
-            <Box
-              display="flex"
-              alignItems="center"
-              justifyContent="spaceBetween"
-            >
-              {file.name}
-              {isUploading && file.checked && <LoadingDots />}
-            </Box>
-          }
-          name={file.id}
-          value={file.id}
-          checked={file.checked}
-          onChange={onCheck}
-        />
-      </CheckboxListItem>
+      <Fragment key={file.id}>
```

Comment:
This was done because we were getting a console warning saying "key is not a prop"